src/Files.App/Filesystem/FolderHelpers.cs

```diff
@@ -2,52 +2,72 @@
 // Licensed under the MIT License. See the LICENSE.
 
 using Files.App.Filesystem.StorageItems;
+using Files.Backend.Helpers;
 using System.IO;
-using static Files.Backend.Helpers.NativeFindStorageItemHelper;
 
 namespace Files.App.Filesystem
 {
+	/// <summary>
+	/// Provides static helper for storage folders.
+	/// </summary>
 	public static class FolderHelpers
 	{
 		public static bool CheckFolderAccessWithWin32(string path)
 		{
-			IntPtr hFileTsk = FindFirstFileExFromApp($"{path}{Path.DirectorySeparatorChar}*.*", FINDEX_INFO_LEVELS.FindExInfoBasic,
-				out WIN32_FIND_DATA _, FINDEX_SEARCH_OPS.FindExSearchNameMatch, IntPtr.Zero, FIND_FIRST_EX_LARGE_FETCH);
+			IntPtr hFileTsk = NativeFindStorageItemHelper.FindFirstFileExFromApp(
+				$"{path}{Path.DirectorySeparatorChar}*.*",
+				NativeFindStorageItemHelper.FINDEX_INFO_LEVELS.FindExInfoBasic,
+				out NativeFindStorageItemHelper.WIN32_FIND_DATA _,
+				NativeFindStorageItemHelper.FINDEX_SEARCH_OPS.FindExSearchNameMatch,
+				IntPtr.Zero,
+				NativeFindStorageItemHelper.FIND_FIRST_EX_LARGE_FETCH);
+
 			if (hFileTsk.ToInt64() != -1)
 			{
-				FindClose(hFileTsk);
+				NativeFindStorageItemHelper.FindClose(hFileTsk);
+
 				return true;
 			}
+
 			return false;
 		}
 
 		public static async Task<bool> CheckBitlockerStatusAsync(BaseStorageFolder rootFolder, string path)
 		{
 			if (rootFolder?.Properties is null)
-			{
 				return false;
-			}
+
 			if (Path.IsPathRooted(path) && Path.GetPathRoot(path) == path)
 			{
 				IDictionary<string, object> extraProperties =
 					await rootFolder.Properties.RetrievePropertiesAsync(new string[] { "System.Volume.BitLockerProtection" });
-				return (int?)extraProperties["System.Volume.BitLockerProtection"] == 6; // Drive is bitlocker protected and locked
+
+				return (int?)extraProperties["System.Volume.BitLockerProtection"] == 6; // Drive is BitLocker protected and locked
```

Comment:
Can you move the comment above the line?

Reply:
Fixed